spec/operations/fts.coffee

```diff
@@ -0,0 +1,44 @@
+operation = require '../../lib/operations/fts'
+
+describe "/off", ->
```

Comment:
describe string refers to wrong endpoint

Reply:
Fixed